source/code/plugins/collectd_lib.rb

```diff
@@ -27,7 +27,14 @@ def transform(record, hostname)
       data_items = []
       data_info = {}
       data_info["Timestamp"] = record["Timestamp"]
-      data_info["Host"] = hostname
+
+      host = record["host"] if !record["host"].nil?     
```

Comment:
Why "host" starts with lower case in record["host"]  but upper case in data_info? Are these case sensitive?